client/src/LayoutComponents/CohortDetails/CohortDetails.js

```diff
@@ -7,38 +7,54 @@ import axios from 'axios';
 class CohortDetails extends Component {
   state = {
     cohortInfo: [],
+    cohortstd: [],
+    cohortMentor: [],
   };
   componentDidMount = () => {
+    const requestOne = axios.get(
+      `/api/cohortinfo/${this.props.match.params.cohortID}`
+    );
+    const requestTwo = axios.get(
+      `/api/cohortstd/${this.props.match.params.cohortID}`
+    );
+    const requestThree = axios.get(
+      `/api/cohortMentor/${this.props.match.params.cohortID}`
+    );
+
     axios
-      .get(`/api/cohortinfo/${this.props.match.params.cohortID}`)
-      .then((result) => {
-        const resultarray = result.data;
-        this.setState({
-          cohortInfo: resultarray,
-        });
-      })
-      .catch((error) => error);
+      .all([requestOne, requestTwo, requestThree])
+      .then(
+        axios.spread((...responses) => {
+          this.setState({
+            cohortInfo: responses[0].data,
+            cohortstd: responses[1].data,
+            cohortMentor: responses[2].data,
+          });
+        })
+      )
+      .catch((err) => err);
```

Comment:
You could output the error to the console and then show the user an error using a flag in `this.state` if you wanted to handle errors in this PR as well.

Reply:
i will do it, in new pull request.